site_prism.gemspec

```diff
@@ -12,7 +12,8 @@ Gem::Specification.new do |s|
   s.email       = 'nat@natontesting.com'
   s.homepage    = 'http://github.com/natritmeyer/site_prism'
   s.summary     = 'A Page Object Model DSL for Capybara'
-  s.description = 'SitePrism gives you a simple, clean and semantic DSL for describing your site using the Page Object Model pattern, for use with Capybara'
+  s.description = 'SitePrism gives you a simple, clean and semantic DSL for describing your site.
+SitePrism implements the Page Object Model pattern, through its usage with Capybara.'
```

Comment:
"through its usage with Capybara." this fragment doesn't parse well for me.  I'm not sure that's the correct preposition (I expected "usage of Capybara" or "use with Capybara").  
Are you trying to say that SitePrism is used along-side (together with) Capybara or that SitePrism uses Capybara?

How about "SitePrism implements the Page Object Model pattern on top of Capybara." ?  Or "SitePrism implements the Page Object Model pattern for usage with Capybara."  (I prefer my 1st suggestion.)